pyproject.toml

```diff
@@ -15,13 +15,16 @@ terralab = "terralab.cli:cli"
 [tool.poetry.dependencies]
 python = "^3.12"
 terra-scientific-pipelines-service-api-client = "0.1.7"
```

Comment:
will need to update this version here after merging https://github.com/DataBiosphere/terra-scientific-pipelines-service/pull/164